### ISSUE TITLE
World analyzer showing incorrect events?

### ISSUE THREAD

**Issue by Razzmatazzz** [Razzmatazzz/RemnantSaveGuardian#44]
If the world analyzer is showing events that are confirmed incorrect for your roll, please report it here. Please indicate the world and event the analyzer showed (with a link to the [wiki page](https://remnant2.wiki.fextralife.com)). If a different event appeared in place of that event and was not shown on the analyzer, please indicate which event appeared instead. Also please export your save via the world analyzer after you pick up the item and post the files here. Indicate whether it was your first, second, third, etc. character.

**Comment by sinnoaria** [comment]
Not sure if this counts, but Morrow Parish is the location and I see the Booster Ring. The guardian lists it as being in the Oracle's Refuge (Which is a sub-area of Morrow Parish) and Oracle's Refuge is the closest world stone. Just mentioning it in case it wasn't intentional.

**Comment by Delusionnnn** [comment]
Showing I have Cathedral of Omens in Campaign when I dont. It is only possible to have 4 dungeons (non boss rooms or middle dungeon) in Yaesha and I have 5. Nothing is in its place.

Also showing I have postulants parlor in Losomn when I dont. Nothing is in its place. 

First character.

[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12326300/profile.txt)
[save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12326301/save_0.txt)

**Comment by Razzmatazzz** [comment]
Is it possible the world analyzer is mixing up which world belongs to each character? If you select a different character on the analyzer, do their events correspond to the other character?

**Comment by Delusionnnn** [comment]
> Is it possible the world analyzer is mixing up which world belongs to each character? If you select a different character on the analyzer, do their events correspond to the other character?

No, I dont have any characters with this. I only have one character and one hardcore character that has only one world generated and it is not Yaesha.

**Comment by AndrewSav** [comment]
[Death's Embrace](https://remnant2.wiki.fextralife.com/Death's+Embrace) and [Blood Tinged Ring](https://remnant2.wiki.fextralife.com/Blood+Tinged+Ring) are shown in analyser and Blood Moon Altar. 

Has anyone actually see them there?

It looks like Death's Embrace is sold by Bedel of the Vaunnt and Blood Tinged Ring is a part of the Supply Room event that usually spawns in Lament where you also get Red Widow armour.

**Comment by Razzmatazzz** [comment]
I think those were associated with that event in the initial event/item list AuriCrystal submitted.

**Comment by AndrewSav** [comment]
> I think those were associated with that event in the initial event/item list AuriCrystal submitted.

If no one can corroborate, I'd remove them from the Blood Moon Altar.

**Comment by merb22** [comment]
All adventure modes in Losomn show Losomn: Shattered Gallery.  I ran it 3 times and it was not there (completely cleared world every time).

did analyzer to make a few more adventure modes and they all show Shattered Gallery at the very end of the list, but I'm not convinced they are there.  I will report back after doing some more runs, but so far 0/3

**Comment by Razzmatazzz** [comment]
All locations are determined by the events seen in the analyzer. If the analyzer is showing Shattered Gallery, then it's seeing an event that it thinks takes place in Shattered Gallery. It could be wrong about the location, but I would expect the events themselves to be present.

**Comment by razeedazee** [comment]
@merb22 do you have a screenshot of your checkpoint travel map?

**Comment by DanMartin85** [comment]
Since 1.0.5.0 (i think at least) the analyzer has been showing the Tal'Ratha world boss yet i dont have that story campaign for N'Erud.

![2023-08-24_9-14-19](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/19556923/e64b3c04-f978-44ad-92e8-8f98f66fd3a0)

**Comment by crackedmind** [comment]
https://remnant2.wiki.fextralife.com/Decayed+Claws Missing from Root Earth

**Comment by Trandel81** [comment]
First of all, absolutely fantastic app, and thank you for it! I keep getting notified that I have the Huntress boss and Briella's Garden in my Losomn world but It is never there.

https://remnant2.wiki.fextralife.com/The+Huntress


1st Character
[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12483880/profile.txt)
[save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12483881/save_0.txt)
[save_1.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12483882/save_1.txt)

**Comment by ethaldeman** [comment]
> First of all, absolutely fantastic app, and thank you for it! I keep getting notified that I have the Huntress boss and Briella's Garden in my Losomn world but It is never there.
> 
> https://remnant2.wiki.fextralife.com/The+Huntress
> 
> 1st Character [profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12483880/profile.txt) [save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12483881/save_0.txt) [save_1.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12483882/save_1.txt)
[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12561320/profile.txt)
[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12561323/profile.txt)
[save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12561324/save_0.txt)


I too am having this issues. My world had both the Dran Oracle and Clock tower Event, so the Huntress Boss could not spawn.

https://remnant2.wiki.fextralife.com/The+Huntress
https://remnant2.wiki.fextralife.com/Oracle+of+the+Dran
https://remnant2.wiki.fextralife.com/The+Clock+Tower+Event

[save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12561331/save_0.txt)
[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12561332/profile.txt)

**Comment by ethaldeman** [comment]
The [Shock Device](https://remnant2.wiki.fextralife.com/Shock+Device) appeared in my world, and I believe is a static item in the Morrow Parish; Losom, but was not listed as an item present.

First Character and Adventure
[save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12679025/save_0.txt)
[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12679037/profile.txt)

**Comment by ethaldeman** [comment]
The [Burden of the Warlock](https://remnant2.wiki.fextralife.com/Burden%20of%20the%20Warlock) and FieryGraveyard were listed as in Losomn: Malefic Palace, but where actually in Losomn: Cotton's Kiln.

First Character and Adventure
[save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12721102/save_0.txt)
[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/12721103/profile.txt)

**Comment by Skynetexe** [comment]
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/117482613/fdac7589-75c5-4313-933a-7a1fb27c0401)
I just picked up the following amulet. It was located in "Pathway of the Fallen" not in the walk of remembrance 
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/117482613/17e6c3c4-0c17-4dc8-bfce-ae69f7d58335)

**Comment by SubjectToChange** [comment]
Rolled a campaign, RSG said it'd feature a Legacy Protocol (https://remnant2.wiki.fextralife.com/Legacy%20Protocol) world drop in N'Erud Train station dungeon but it was something different, sadly I didn't record what it was before picking up the dupe.
[profile.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13920532/profile.txt)
[save_0.txt](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13920533/save_0.txt)

Only character I have currently.

**Comment by mbsff** [comment]
Lithic Signet, Weeping angels event displaying as happeing in the red throne, it occured in the Infested Abyss.
![image](https://github.com/user-attachments/assets/3a35700c-2e8a-4b51-a3d8-f7381d8f0a0e)
![image](https://github.com/user-attachments/assets/e2d65a95-1ff9-4bb7-85a2-7a7ebfa77411)
https://remnant2.wiki.fextralife.com/Living+Stone
[profile - Copy.txt](https://github.com/user-attachments/files/18369047/profile.-.Copy.txt)
[save_0 - Copy.txt](https://github.com/user-attachments/files/18369048/save_0.-.Copy.txt)

**Comment by Eenna** [comment]
Analyzer says this item (Tear of Kaeula) is still in my world when it's not. Possible confusion could come from the fact that I wasn't the one to pick it up: my multiplayer friend picked it up. I'm like 80% sure we both got it on pick up tho.
![Image](https://github.com/user-attachments/assets/cc9fc83e-77ef-4d76-8524-64556d971b4f)
